scss/_base_typography-definitions.scss

```diff
@@ -197,10 +197,10 @@
     @extend %default-text;
     font-variant-caps: all-small-caps;
     letter-spacing: 0.05em;
-    margin-bottom: map-get($sp-after, default-text) - map-get($nudges, p);
+    margin-bottom: map-get($sp-after, p-small-caps) - map-get($nudges, p);
 
     &.u-no-margin--bottom {
-      @extend %u-no-margin--bottom--small;
+      @extend %u-no-margin--bottom--default-text;
```

Comment:
At a glance, I think WTH at this? I understand `--small` might mean no margin bottom and small screen but now we have no margin only on ... Default text?

Reply:
that’s just fixinga bug. with the small caps in the new variable font, muted heading now uses the same size as p, h5 and h6. to avoid having all those in the nudges and space-after maps, they are grouped under the key default-text. So since the font size of the muted heading changed, we’re updating the calculation to use the appropriate map keys.